src/daft-io/src/lib.rs

```diff
@@ -469,6 +469,25 @@ impl Runtime {
         Arc::new(Self { runtime })
     }
 
```

Comment:
discussed offline but let's move this Runtime to a common crate and have the static impl live there.

Ideally the compute pool and io pool can just be different instances of the same Runtime struct.